pages/DSNP/Announcements.md

```diff
@@ -4,16 +4,11 @@ name: Announcements
 route: /DSNP/Announcements
 ---
 
-
-# TODO
-
-I took out signatures from this level. Do we still need them for DSNP 1.0?
-
-
 # Announcements Overview
 
 Announcements are content or reference to content that communicate new user activity to the rest of the network.
 Announcements are associated with an [Identifier](/DSNP/Identifiers) which can be validated as creator of the announcement.
+Announcement validation starts with the [signature](/DSNP/Signatures) of the announcement.
```

Comment:
Now that I read this line again I think we should revisit this line. I'm thinking re-word it and then add a section below about `## Announcement Validation`? Thoughts @saraswatpuneet ?

Reply:
@wilwade  I agree, let me try it out  then we can discuss tomorrow/pair